tests/integration/telemetry/stats/prometheus/http/customizemetrics/testdata/stats_filter_update.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: stats-filter-update
+spec:
+  workloadSelector:
+    labels:
+      app: server
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_INBOUND
+        listener:
+          filterChain:
+            filter:
+              name: envoy.http_connection_manager
+              subFilter:
+                name: envoy.router
+      patch:
+        operation: INSERT_BEFORE
+        value:
+          name: istio.stats
+          typed_config:
+            '@type': type.googleapis.com/udpa.type.v1.TypedStruct
+            type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+            value:
+              config:
+                configuration:
+                  '@type': type.googleapis.com/google.protobuf.StringValue
+                  value: |
```

Comment:
This can be configured using istioctl: https://istio.io/latest/docs/tasks/observability/metrics/customize-metrics/#enable-custom-metrics

Reply:
then it would be better to keep it as separate test suite because it requires different istioctl setup. Though we can do that in the test runtime as well but seems like it would be better to do it at setup time?

Reply:
I see, yeah let's keep it in different test setup then.